src/router/index.js

```diff
@@ -51,7 +51,7 @@ const routes = [
     component: Employees
   },
   {
-    path: '/customers/:id',
+    path: '/customers/:eid',
```

Comment:
eid stands for employee_id it can't be used for customers. Need to reimplement Search.